app/src/main/java/com/example/mac/chartr/activities/PostTripActivity.java

```diff
@@ -0,0 +1,98 @@
+package com.example.mac.chartr.activities;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Switch;
+import android.widget.TextView;
+
+
+import com.example.mac.chartr.R;
+
+
+import org.w3c.dom.Text;
+
+
+import java.sql.Time;
+import java.util.Date;
+
+import com.example.mac.chartr.R;
+
+
+public class PostTripActivity extends AppCompatActivity {
+    private static final String TAG = MainActivity.class.getSimpleName();
+
+    private EditText inStartLocation;
+    private EditText inEndLocation;
+    private EditText inDepartureDate;
+    private EditText inReturnDate;
+    private EditText inDepartureTime;
+    private EditText inReturnTime;
+    private Switch inCanPickUp;
+    private TextView inNumSeats;
+    private Switch inAllowSmoking;
+    private Switch inIsQuiet;
+
+    private String startLocation;
+    private String endLocation;
+    private Date departureDate;
+    private Date returnDate;
+    private Time departureTime;
+    private Time returnTime;
+    private boolean canPickUp;
+    private int numSeats;
+    private boolean allowSmoking;
+    private boolean isQuiet;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_post_trip);
+    }
+
+    public void incrementSeats(View view){
+        inNumSeats = (TextView) findViewById(R.id.seatValue);
+        numSeats = Integer.parseInt(inNumSeats.getText().toString());
+        if(numSeats < 4) {
+            inNumSeats.setText(String.valueOf(numSeats + 1));
+        }
+    }
+
+    public void decrementSeats(View view){
+        inNumSeats = (TextView) findViewById(R.id.seatValue);
+        numSeats = Integer.parseInt(inNumSeats.getText().toString());
+        if(numSeats > 1) {
+            inNumSeats.setText(String.valueOf(numSeats - 1));
+        }
+    }
+
+    public void postTrip(View view) {
+
+        /**
+         * Sample http request with volley
+        // Instantiate the RequestQueue.
+        RequestQueue queue = Volley.newRequestQueue(this);
+        String url ="https://99sepehum8.execute-api.us-east-2.amazonaws.com/prod/user/*?email=joe.smitty@gmail.com";
```

Comment:
I know this is a demo, but uh, why are you trying to post a trip to the "/user" endpoint? Trips go to the "/trip" endpoint...

Reply:
That was just to test that everything worked as it should.  User was working before trip, so that's what I used for testing.

Reply:
Ah, gotcha. That makes sense